packages/manager-components/src/components/templates/dashboard/dashboard.component.tsx

```diff
@@ -1,47 +1,51 @@
 import React from 'react';
-
 import { Headers, HeadersProps } from '../../content';
-
-import { Links, LinksProps } from '../../typography';
+import { Description, LinkType, Links, Subtitle } from '../../typography';
 import { PageLayout } from '../layout/layout.component';
+
 export interface DashboardLayoutProps {
   breadcrumb?: React.ReactElement;
   content?: React.ReactElement;
   header?: HeadersProps;
-  linkProps?: LinksProps;
+  message?: React.ReactElement;
```

Comment:
We can have a requirement to display multiple messages and its type can be varied as well among info/warning/error. I think no need to include message instead we can provide a slot to fill it with children.

Reply:
Yes it's fine we can put multiple messages in this slot using fragment :

```
<>
  <OsdsMessage>{message1}</OsdsMessage>
  <OsdsMessage>{message2}</OsdsMessage>
</>
```